NewSpring/corinth
# feat: Remove an answer

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds the ability to remove an answer to a prayer.

![remove-answer](https://user-images.githubusercontent.com/3229463/81083651-d796b800-8ec2-11ea-8138-05281e30f40a.gif)

### How do I test this PR?

- Open up `My Prayers`
- Find a prayer that doesn't have an answer
- Tap on the ellipsis icon and choose "Mark as Answered"
- On the answer prayer screen you should NOT see an option for "Remove answer"
- Enter an answer and tap "Save answer"
- The answer prayer screen should close and take you back to My Prayers. The answer to the prayer should show on the prayer.
- Tap the ellipsis icon again and choose "Edit Answer".
- On the answer prayer screen there should now be an option for "Remove answer"
- Tap that and tap again on the confirmation.
- You should be sent back to the My Prayers list and the answer to that prayer should be gone.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
